File: Testing/Code for Testing/Early Basic Syntax Testing/constant/test_constant.py
from dataclasses import dataclass, field

def _ul_sort(arr, n): arr[:n] = sorted(arr[:n])

class _ul_Exception(Exception):
    def __init__(self, type_name, message):
        self.type_name = type_name
        self.message = message
        super().__init__(message)

MAX = 100
VERSION = '1.0.0'
PI = 3.14

def main():
    print(MAX)
    print(VERSION)
    print(PI)
    return 0


if __name__ == "__main__":
    main()

File: Testing/Code for Testing/Early Basic Syntax Testing/constant/test_constant.js

const MAX = 100;
const VERSION = "1.0.0";
const PI = 3.14;
function main() {
    console.log(MAX);
    console.log(VERSION);
    console.log(PI);
    return 0;
}

function __ul_sort(arr, n) { const sub = arr.slice(0, n).sort((a, b) => a < b ? -1 : a > b ? 1 : 0); for (let i = 0; i < n; i++) arr[i] = sub[i]; }

main();
